### DISCUSSION TITLE
Can I currently attend this meeting in person? wont update

### DISCUSSION THREAD

**Discussion by COMPUMEC** [code4recovery/12-step-meeting-list#459] [General]
**Describe the bug**
Under location information I change "Can I currently attend this meeting in person?" from "no" to "yes" and click update. The page refreshes and the "yes" goes back to "no". It wont update.

**To Reproduce**
Under location information I change "Can I currently attend this meeting in person?" from "no" to "yes" and click update. The page refreshes and the "yes" goes back to "no". It wont update.

**Expected behavior**
I would expect changing from no to yes and clicking update would hold and not revert to no.

**Screenshots**
NA

**Desktop (please complete the following information):**
Win 10
Chrome
91.0.4472.164

**Smartphone (please complete the following information):**

**Additional context**

**Comment by tech2serve** [comment]
Please provide screen caps of the Location block on the admin edit page before and after you click on the "Yes". 

My guess is that you have an approximate address entered, and the address needs to be specific (street, city, state) in order for a meeting to be attended in person. However, in that case, you should see this error message display on your screen:

<img width="772" alt="image" src="https://user-images.githubusercontent.com/31904316/126051156-cf1fffb5-1582-4042-8009-f5434b357b01.png">

If the address you have entered _is_ specific, please share it. I'm curious if perhaps there is an issue geocoding the address.

And if none of the above clears up the issue, please open of a dev console (search the Internet for how to do with with your choice of browser) and look for any errors when you load the meeting edit page or click on the Yes.

Keep us posted, please.

**Reply by tech2serve** [reply]
Hi @COMPUMEC--

I'm glad you've narrowed it down to the one location. 

There are still some troubleshooting steps here for you to take and report back on. Especially opening a dev/javascript console and then entering the address, followed by tab. That should tell us if any errors are being thrown.

BTW, I was able to create a meeting on my dev site with the _37 Skowhegan Rd, Fairfield, ME 04937_ and I did not experience any issues. 

So, my guess is that something may have gotten hung up in the cache. Posting screen caps of what you have on the admin edit meeting page will help narrow this down.

Yis,

**Comment by COMPUMEC** [comment]
Hello,

Please look at this video and tell me what you think.

I deleted this meeting waited 24 hours and went in and tried to re create it.

https://dl.dropboxusercontent.com/s/rgzevuf91vxaigi/Meetings%20%E2%80%B9%20Central%20Service%20Office%20%E2%80%94%20WordPress%20-%20Google%20Chrome%202021-07-21%2013-18-45.mp4?dl=0

Thank you,
Bob-

**Reply by tech2serve** [reply]
The video was very helpful. I think @joshreisner is probably onto something. Given that the address seems correct in Google, not sure we need to have it hard coded any more. We’ll discuss internally and if it can be removed, we’ll start that and then your site should geocode it fresh.

Thanks for your patience.

**Comment by joshreisner** [comment]
@tech2serve this address (and a few others) are currently defined in [variables.php](https://github.com/code4recovery/12-step-meeting-list/blob/master/includes/variables.php#L165-L170) - my guess is that the geocoder is missing the `approximate = false` because it's hard-coded

**Reply by COMPUMEC** [reply]
So what's to be done. 
I just today I tried to add it without success.
Is this a dead end or is there a resolve?
Thank you,
Bob-

**Reply by joshreisner** [reply]
Hey Bob, the group talked about this issue this weekend on our biweekly call. A fix should be coming in the next version of the plugin. I don't have a firm expectation on when that would happen, but likely in the next few weeks.

If you can hold on to then great. Otherwise if you want to edit a file in your plugin via FTP or some other method (not via WordPress itself) I can show you how you'd be able to do that, although there is some risk involved.

**Reply by brianw-area24** [reply]
Yes, I was able to recreate the issue on my test site, and it's definitely related to the address being hard-coded in the plugin. The fix we're working on should fix this issue, at least it does in my test environment.

**Reply by COMPUMEC** [reply]
Hey Josh it has been a while. Hope you are well. Keep Trusting & Keep Believing!
Thank you all for the work you do. I will wait on the update.

**Reply by joshreisner** [reply]
thanks bob! brian w is the real mvp now 😄
